templates/index.html

```diff
@@ -308,11 +308,11 @@
             <label class="heading" id="label" for="email">Email</label>
             <div class="input-group margin-bottom-sm">
              <span class="input-group-addon"><i class="fa fa-envelope-o fa-fw"></i></span>
-            <input autofocus="" class="form-control" placeholder="Enter email" name="email" title="Email for sending the ISO" required="" type="email"></div>
+            <input autofocus="" class="form-control" placeholder="Enter email" name="email" title="Email for sending the ISO" required="" type="email" pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)+$" required /> </div>
```

Comment:
This  looks good. If you could provide a link to the source of this pattern, this would give credit.

Reply:
https://github.com/fossasia/meilix-generator/issues/134

Reply:
https://stackoverflow.com/a/26272713